althea_kernel_interface/src/lib.rs

```diff
@@ -81,7 +82,7 @@ impl KernelInterface {
                 "-A",
                 "INPUT",
                 "-s",
-                &format!("{}", source_neighbor),
+                &format!("{}", source_neighbor.to_hex_string()),
```

Comment:
I'm surprised that `format!` doesn't do this automatically with the `to_string` trait or whatever.

Reply:
@jtremback it's the MAC package's gimmick, default format is `aa-bb-cc-dd-ee-ff` when `aa:bb:cc:dd:ee:ff` is used more often and that's what `to_hex_string` formats it to